### PR TITLE
Update username regex to string literal with escaped -

### PR DESCRIPTION
**Pull request by LogicalPhallacy**
**Changes**
The username regex was hard to parse because of the double \\ and the unescaped -

Even though it worked this minor fix makes it clearer what it is doing

### CODE REVIEW

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 1]
```diff
@@ -216,10 +216,10 @@ public void Initialize()
 
         public static bool IsValidUsername(string username)
         {
-            //This is some regex that matches only on unicode "word" characters, as well as -, _ and @
-            //In theory this will cut out most if not all 'control' characters which should help minimize any weirdness
+            // This is some regex that matches only on unicode "word" characters, as well as -, _ and @
+            // In theory this will cut out most if not all 'control' characters which should help minimize any weirdness
             // Usernames can contain letters (a-z + whatever else unicode is cool with), numbers (0-9), dashes (-), underscores (_), apostrophes ('), and periods (.)
```
Comment (LeoVerto) [inline Emby.Server.Implementations/Library/UserManager.cs]: Can you mention that "@" is a valid character in this comment? (And also the error message below).

File: Emby.Server.Implementations/Library/UserManager.cs [hunk 2]
```diff
@@ -216,10 +216,10 @@ public void Initialize()
...
-            return Regex.IsMatch(username, "^[\\w-'._@]*$");
+            return Regex.IsMatch(username, @"^[\w\-'._@]*$");
```
Comment (LeoVerto) [inline Emby.Server.Implementations/Library/UserManager.cs]: A lot more readable, good job!

File: Emby.Server.Implementations/Library/UserManager.cs (hunk 1 above)
Comment (joshuaboniface) [reply Emby.Server.Implementations/Library/UserManager.cs]: Yes please! I'm happy that this will finally work so I can use full email addresses for usernames.

Comment (joshuaboniface) [reply Emby.Server.Implementations/Library/UserManager.cs]: // Usernames can contain letters (a-z + whatever else unicode is cool with), numbers (0-9), at-signs (@), dashes (-), underscores (_), apostrophes ('), and periods (.)
